docs/import/add_disaster.js

```diff
@@ -0,0 +1,289 @@
+import {eeStatePrefixLength, legacyStateDir} from '../ee_paths.js';
+import {getFirestoreRoot} from '../firestore_document.js';
+
+export {enableWhenReady, SENTINEL_NEW_DISASTER_VALUE, toggleState};
+// Visible for testing
+export {
+  addDisaster,
+  createAssetPickers,
+  createOptionFrom,
+  disasters,
+  emptyCallback,
+  getAssetsFromEe,
+  stateAssets,
+  writeNewDisaster,
+};
+
+// Currently a map of disaster name to states. This pulls once on firebase
+// authentication and then makes local updates afterwards so we don't need to
+// wait on firebase writes to read new info.
+const disasters = new Map();
+
+// Map of state to list of known assets
+const stateAssets = new Map();
+
+const SENTINEL_OPTION_VALUE = '...';
```

Comment:
Do we still need this?

Reply:
I was using it for the upload additional assets option. Now we don't anymore.